docs/advanced-topics/networkobject-parenting.md

```diff
@@ -0,0 +1,173 @@
+---
+id: networkobject-parenting
+title:  NetworkObject Parenting
+description: A `NetworkObject` reparenting solution within Netcode for GameObjects (Netcode)  to help developers with synchronizing transform parent-child relationships of `NetworkObjects`.
+
+---
+
+:::important Opt-OUT
+This feature is behind a bool flag that can be toggled on the `NetworkObject` inspector UI. It will be enabled by default but you can can opt-out from it if you want to implement your own solution
```

Comment:
```suggestion
This feature is behind a bool flag that can be toggled on the `NetworkObject` inspector UI. It will be enabled by default but you can opt-out from it if you want to implement your own solution
```